### ISSUE TITLE
is it possible to use kapacitor without influxdb?

### ISSUE THREAD

**Issue by Jaeyo** [influxdata/kapacitor#2401]
I found `/write` api, and maybe InfluxDB server send metric into this api with subscriptions.
InfluxDB also has same api `/write`, and Telegraf send metric into `/write` api.
So, is it possible just Telegraf send metric to Kapacitor with `InfluxDB Output Plugin` and using Stream Node to catch alert?

**Comment by m4ce** [comment]
yes, it's possible and it's actually the way to do it in influx 2.0. It's also documented here -> https://docs.influxdata.com/influxdb/cloud/tools/kapacitor/.
